pyproject.toml

```diff
@@ -1,3 +1,7 @@
+[build-system]
+requires = ["jupyter_packaging~=0.5.0", "setuptools>=40.8.0", "wheel"]
```

Comment:
Wondering if we need to stick to `0.5.x` for jupyter packaging here.

Or if it could be bumped to a more recent version, for example: `jupyter_packaging~=0.12.0`

Reply:
Also of note, we will probably soon be able to drop `jupyter-packaging` and switch to `hatch` (in a later PR):

https://github.com/jupyterlab/team-compass/issues/146